Automattic/jetpack
# [Plugin] Backup: Change button border and updated margins for sections

Related tasks:
as-1200917815086373-1200917815086391
as-1200917815086373-1200917815086395

#### Changes proposed in this Pull Request:
* Changes the 'See your backups' border color from blue to black
* Updates section margins to match following image
![image](https://user-images.githubusercontent.com/1992658/132714101-fe79e562-c29c-4c26-ab29-30d03a0b4c5d.png)

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Install this branch of the backup plugin with Jetpack Beta
* Purchase a plan and wait for a backup to finish (or install on a site that already has backups)
* Confirm that the `See all your backups` button has black border like the below image
<img width="304" alt="Screen Shot 2021-09-09 at 11 20 17 AM" src="https://user-images.githubusercontent.com/1992658/132714336-751d1120-659a-4223-bff3-8b2fe1346fe8.png">
Previously the border was blue as below

![image](https://user-images.githubusercontent.com/1992658/132715488-e82439dc-a982-42a3-8861-76a17d18ac61.png)

* Confirm that the section spacing matches the first image in the changes section above.
